### DISCUSSION TITLE
problem with some python modules !!!

### DISCUSSION THREAD

**Discussion by Y-AMDIAZ** [omnetpp/omnetpp#1106] [Q&A] [answered]
I want to download the last version of omnet++ but I had a probleme with the instalation , it gives me that I have a messing modules of python or I dont know can some one help me please how can I solve this problem 
Omnet++6.0.1
windows 11 

<img width="865" alt="image" src="https://github.com/omnetpp/omnetpp/assets/129278941/307a5acc-5321-475b-9592-bc02c3000603">

**Comment by rhornig** [comment]
This usually happens because you already have Python installed and mingwenv picks up the system installation and adds that to the PATH. Remove the PYTHONPATH environament variable from the system environment (or add `export PYTHONPATH=` to the setenv script.

**Comment by Y-AMDIAZ** [comment]
Should I delete the line or what? How can I add an export, and I found export PYTHONPATH= in the file before?
<img width="708" alt="image" src="https://github.com/omnetpp/omnetpp/assets/129278941/937f106d-b153-43fd-a572-3a9530691f5d">
<img width="863" alt="image" src="https://github.com/omnetpp/omnetpp/assets/129278941/ff0ac311-5342-4df1-906b-a963266fbff8">

**Comment by rhornig** [comment, marked as answer]
Actually omnetpp-6.0.2 contains this fix, so you are probably better off using that version. (or wait for the upcoming 6.0.3 version)

...or you can manually add the the # Windows/MinGW specific settings if/then block.
